api.md

```diff
@@ -431,12 +434,104 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [`chunk_relation_size`](#chunk_relation_size)
 function.
 
-**Space partitions**: In most cases, it is advised for users not to use
+**Space partitions:** In most cases, it is advised for users not to use
 space partitions. The rare cases in which space partitions may be useful
 are described in the [add dimension][] section.
 
 ---
 
+## create_distributed_hypertable() [](create_distributed_hypertable)
+
+Creates a TimescaleDB hypertable distributed across a multinode
+environment.  This is very similar to [`create_hypertable`](#create_hypertable),
+and in fact has all of the same required and optional arguments.  The
+main difference is that this command will default the
+`replication_factor` optional parameter to a value of 1, and will
```

Comment:
Do not go into replication_factor here. IMO, there's generally too much mention of replication_factor here considering replication is not a feature we're launching at this point.

I would suggest something simple like this:


"Creates a distributed hypertable across multiple data nodes. Use this function in place of `create_hypertable` when creating distributed hypertables."

Reply:
Modified as you suggested.  We should look at just removing replication_factor from the exposed API here for the public beta, unless we're planning on building up more support by then.